### PR TITLE
nicotine+: update to 2.0.1 (port to python3)

### PR DESCRIPTION
**Pull request by st3r4g**
Ported to python3

### CODE REVIEW

**Comment by st3r4g** [comment]
Could drop `librsvg` in `depends` after https://github.com/void-linux/void-packages/pull/23530

File: srcpkgs/nicotine+/template [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-pycompile_module="pynicotine"
+wrksrc="nicotine-plus-${version}"
+build_style=python3-module
+hostmakedepends="python3"
+depends="python3-gobject python3-mutagen gtk+3 librsvg"
 short_desc="Soulseek music-sharing client written in Python"
 maintainer="doggone <doggone@airmail.cc>"
 license="GPL-3.0-or-later"
-homepage="http://www.nicotine-plus.org/"
+homepage="https://nicotine-plus.github.io/nicotine-plus"
```
Comment (mathiascode) [inline srcpkgs/nicotine+/template:13]: Even though the nicotine-plus.org domain now redirects, https://nicotine-plus.org/ is the preferred URL.

File: srcpkgs/nicotine+/template [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-wrksrc="nicotine-plus-${version}"
 archs=noarch
-build_style=python2-module
-hostmakedepends="python-gobject2-devel"
-depends="pygtk mutagen librsvg"
-pycompile_module="pynicotine"
+wrksrc="nicotine-plus-${version}"
+build_style=python3-module
+hostmakedepends="python3"
+depends="python3-gobject python3-mutagen gtk+3 librsvg"
```
Comment (mathiascode) [inline srcpkgs/nicotine+/template:9]: Consider introducing python3-miniupnpc as a required dependency, if possible. https://github.com/Nicotine-Plus/nicotine-plus/tree/b98790de2b6b900c2450ff43311ee7683497c7db#dependencies

**Comment by mathiascode** [comment]
Thanks for updating!

File: srcpkgs/nicotine+/template (hunk 2 above)
Comment (st3r4g) [reply srcpkgs/nicotine+/template]: all done, thanks!

**Comment by ahesford** [comment]
GTK3 now depends on `librsvg` if you still want to drop the dep here.

**Comment by mathiascode** [comment]
Nicotine+ 2.0.1 has been released.

**Pull request merged**